README.md

```diff
@@ -0,0 +1,210 @@
+# SCT - Scylla Cluster Tests
+
+SCT tests are designed to test Scylla database on physical/virtual servers under high read/write load.
+Currently, the tests are run using built in unittest
+These tests automatically create:
+
+* Scylla clusters - Run Scylla database
+* Loader machines - used to run load generators like cassandra-stress
+* Monitoring server - uses official Scylla Monitoring repo_ to monitor Scylla clusters and Loaders
+
+## Quickstart
+
+```bash
+# Ask your AWS account admin to create a user and access key for AWS) and then configure AWS
+
+> aws configure
+AWS Access Key ID [****************7S5A]:
+AWS Secret Access Key [****************5NcH]:
+Default region name [us-east-1]:
+Default output format [None]:
+
+# Install hydra (docker holding all requirements for running SCT)
+sudo ./install-hydra.sh
+```
+
+### Run a test
+
+Example running test using Hydra using `test-cases/PR-provision-test.yaml` configuration file
+
+on AWS:
+```bash
+hydra "run-test longevity_test.LongevityTest.test_custom_time --backend aws --config test-cases/PR-provision-test.yaml"
+```
+
+on AWS using SCT Runner:
+
+```bash
+ hydra create-runner-instance --cloud-provider <cloud_name> -r <region_name> -z <az> -t <test-id> -d <run_duration>
+ hydra --execute-on-runner <runner-ip|`cat sct_runner_ip> "run-test longevity_test.LongevityTest.test_custom_time --backend aws --config test-cases/PR-provision-test.yaml"
+```
+
+on GCE:
```

Comment:
in 'backends' section we already have some examples.

Reply:
I've remove the backend section,
I think maybe just to list the backend we support - with a brief sentence on each.